src/applications/my-education-benefits/config/form.js

```diff
@@ -341,7 +341,10 @@ function notGivingUpBenefitSelected(formData) {
 }
 
 function transform(metaData, form) {
-  const submission = createSubmissionForm(form.data);
+  const submission = createSubmissionForm(
+    form.data,
+    VA_FORM_IDS.FORM_22_1990EZ,
```

Comment:
The Form Id is defined in the form configuration (form.js line 363) and available in `form.formId` here, so it doesn't need to be hardcoded here.

Reply:
new commit updates VA FORM object to self form.formId